From: Andy Chi <andy.chi@canonical.com>
Date: Wed, 28 Dec 2022 17:36:37 +0800
Subject: ucm2: sof-soundwire: Update Mic LED settings

It would be better to make this settings as default in
the driver. See PR.

Fixes: https://github.com/alsa-project/alsa-ucm-conf/pull/261
Signed-off-by: Andy Chi <andy.chi@canonical.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/sof-soundwire/sof-soundwire.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ucm2/sof-soundwire/sof-soundwire.conf b/ucm2/sof-soundwire/sof-soundwire.conf
index f59aa7d..ab27164 100644
--- a/ucm2/sof-soundwire/sof-soundwire.conf
+++ b/ucm2/sof-soundwire/sof-soundwire.conf
@@ -70,5 +70,6 @@ If.mic_init_rt715 {
 	}
 	True.BootSequence [
 		cset "name='PGA5.0 5 Master Capture Switch' 1"
+		sysw "-/class/sound/ctl-led/mic/card${CardNumber}/detach:PGA5.0 5 Master Capture Switch"
 	]
 }
